packages/scripts/README.md

```diff
@@ -0,0 +1,125 @@
+# @plone/scripts package
+
+This package is a library of scripts that are useful to automate certain tasks when developing Volto core or Volto projects.
+
+## i18n
+
+It scans and detects i18n messages from the code and add them to the i18n machinery.
+
+See https://6.dev-docs.plone.org/volto/recipes/i18n.html for more information.
+
+This script is installed in the `.bin` directory and can be called via `yarn i18n` or directly in the `scripts` `package.json` part.
```

Comment:
Just a check, did you mean `.bin` as an invisible directory, or perhaps `./bin` as being in the working directory?